EnvVars/newrelic.cmd

```diff
@@ -36,7 +39,7 @@ IF %NR_ERROR_LEVEL% EQU 0 (
 	:: WEB ROLES : Restart the service to pick up the new environment variables
 	:: 	if we are in a Worker Role then there is no need to restart W3SVC _or_
 	:: 	if we are emulating locally then do not restart W3SVC
-	IF "%IsWorkerRole%" EQU "false" IF "%EMULATED%" EQU "false" (
+	IF "%IsWorkerRole%" EQU "false" (

```

Comment:
This functionally has been moved up higher in the script to short circuit earlier in the process.  see [line 4](https://github.com/newrelic/newrelic-azure-environment-configuration/blob/9c897845849026d941c9edbf75d38c27b202a8e2/EnvVars/newrelic.cmd#L4) in the cmd